app/templates/app/analytics.html

```diff
@@ -35,99 +35,175 @@
-							<div class="icon"><i class="icon-list-1"></i></div>
-							<div class="name"><strong class="text-uppercase">Average # of Items Per Donation</strong><span>Last 30 days</span>
+							<div class="icon"><i class="icon-bill"></i></div>
+							<div class="name"><strong class="text-center">AVG Items / Donation</strong><span>Total</span>
 								<div id="avgItemPerDonation" class="count-number">70</div>
 							</div>
 						</div>
 					</div>
+					<div class="col">
+						<div class="wrapper count-title d-flex">
+							<div class="icon"><i class="icon-bill"></i></div>
+							<div class="name"><strong class="text-center">AVG Value / Item</strong><span>Total</span>
+								<div id="avgValuePerItem" class="count-number">70</div>
+							</div>
+						</div>
+					</div>
 				</div>
 			</div>
+			</div>
 		</section>
 		<section class="dashboard-header section-padding">
 			<div class="container-fluid">
-				<div class="card chart">
-					<h2 class="display h4">Donation Items by Location</h2>
-					<p> Monthly Donations of Different Locations </p>
-					<div class="bar-chart">
-						<canvas id="locationBarChart" class="chart-bar"></canvas>
+				<div class="row">
+					<div class="col">
+						<div class="card chart">
+							<h2 class="display h4">Donation Item by Location</h2>
+							<p> The number of Donation Item of Different Locations </p>
+							<div class="bar-chart">
+								<canvas id="locationBarChart" class="chart-bar"></canvas>
+							</div>
+						</div>
 					</div>
-				</div>
-				<div class="card chart">
-					<h2 class="display h4">Change of the donation Item</h2>
-					<p>The number of donation items within last one year.</p>
-					<div class="line-chart">
-						<canvas id="lineChart"></canvas>
+					<div class="col">
+						<div class="card chart">
+							<h2 class="display h4">Change of the donation Item</h2>
+							<p> The number of donation items within specified dates </p>
+							<div class="line-chart">
+								<canvas id="itemNumberLineChart"></canvas>
+							</div>
+						</div>
 					</div>
 				</div>
-				<div class="card chart width-50-pct">
-					<h2 class="display h4">The Percentage of the Type of Items</h2>
-					<p>The proportion of items donated by the individual.</p>
-					<div class="pie-chart">
-						<canvas id="pieChart" height="300px"></canvas>
+				<div class="row">
+					<div class="col">
+						<div class="card chart">
+							<h2 class="display h4">Change of the Donation Value</h2>
+							<p> The value of Donation Item within specified dates </p>
+							<div class="line-chart">
+								<canvas id="dateValueLineChart"></canvas>
+							</div>
+						</div>
+					</div>
+					<div class="col">
+						<div class="card chart">
+							<h2 class="display h4">The Percentage of the Type of Items</h2>
+							<p> The proportion of items donated by the individual </p>
+							<div class="pie-chart align-self-center">
+								<canvas id="pieChart" height="300px"></canvas>
```

Comment:
If we are gonna have better id names like other places, we should also changes this.